cobigen-eclipse/cobigen-eclipse/src/com/devonfw/cobigen/eclipse/workbenchcontrol/handler/HealthCheckHandler.java

```diff
@@ -28,13 +28,30 @@ public Object execute(final ExecutionEvent event) throws ExecutionException {
 
     MDC.put(InfrastructureConstants.CORRELATION_ID, UUID.randomUUID().toString());
 
+    checkMonolithicTemplatesException(event);
     try {
-      new HealthCheckDialog().execute();
+      new HealthCheckDialog().executeWithMonolithicTemplates();
```

Comment:
I am wondering to all the code that it has been rewritten towards only focusing on monolithic configuration generation. Why?